diff --git a/dist/render/utils/css/get-matching-global-rules-for-element.js b/dist/render/utils/css/get-matching-global-rules-for-element.js
index a18feded8b5b6ac07b4fd76c897d91f2954a629d..ab7be1081ae6faf4804c0dd18700302dbc6523b0 100644
--- a/dist/render/utils/css/get-matching-global-rules-for-element.js
+++ b/dist/render/utils/css/get-matching-global-rules-for-element.js
@@ -1,4 +1,3 @@
-import { Declaration } from 'postcss';
 // Properties from * selector that should always be applied (layout-critical)
 const ALWAYS_APPLY_PROPS = ['box-sizing', 'margin'];
 // Properties from * selector that are conditionally applied based on element classes
